common/data_with_io.ksy

```diff
@@ -0,0 +1,11 @@
+meta:
+  id: data_with_io
```

Comment:
In my specs I usually call this stuff a `placeholder`.

Reply:
I'm open to suggestions for better names here :) But I don't think `placeholder` is a good term for this... To me, "placeholder" sounds like a part that hasn't been specced/reversed yet, or a field that is not yet used.